grafana/provider.go

```diff
@@ -210,6 +242,8 @@ type client struct {
 	smURL string
 
 	mlapi *mlapi.Client
+
+	aapi *aapi.Client
```

Comment:
I'd call that `amixrAPI` even if that's a bit more verbose. I can read the others, but aapi is a bit harder

Reply:
I would also suggest `ocapi` or `irmapi` instead.